package.json

```diff
@@ -28,9 +28,18 @@
     ],
     "all": true,
     "cache": true,
-    "check-coverage": true
+    "check-coverage": true,
+    "exclude": [
+      "coverage/**",
+      "test{,s}/**",
+      "**/.{prettier,eslint,mocha}rc.{js,cjs}"
+    ]
   },
   "scripts": {
+    "lint:formatting": "prettier . --check",
```

Comment:
We should add a `"format": "prettier . --write",` command for consistency with the other stylelint repos.